packages/sdk/tests/unit/dataProtectorCore/getGrantedAccess.unit.ts

```diff
@@ -0,0 +1,265 @@
+import { describe, expect, it, jest } from '@jest/globals';
+import { ApiCallError } from 'iexec/errors';
+import { ValidationError } from 'yup';
+import { WorkflowError } from '../../../src/index.js';
+import { getGrantedAccess } from '../../../src/lib/dataProtectorCore/getGrantedAccess.js';
+
+describe('getGrantedAccess', () => {
+  describe('Check validation for input parameters', () => {
+    describe('When given protected data is invalid', () => {
+      it('should throw a yup ValidationError with the correct message', async () => {
+        // --- GIVEN
+        const invalidProtectedDataAddress = '0x123456...';
+
+        await expect(
+          // --- WHEN
+          getGrantedAccess({
+            // @ts-expect-error No need for iexec here
+            iexec: {},
+            protectedData: invalidProtectedDataAddress,
+          })
+          // --- THEN
+        ).rejects.toThrow(
+          new ValidationError(
+            'protectedData should be an ethereum address or a ENS name'
+          )
+        );
+      });
+    });
+
+    describe('When given authorizedApp is invalid', () => {
+      it('should throw a yup ValidationError with the correct message', async () => {
+        // --- GIVEN
+        const invalidAuthorizedAppAddress = '0x123456...';
+
+        await expect(
+          // --- WHEN
+          getGrantedAccess({
+            // @ts-expect-error No need for iexec here
+            iexec: {},
+            authorizedApp: invalidAuthorizedAppAddress,
+          })
+          // --- THEN
+        ).rejects.toThrow(
+          new ValidationError(
+            'authorizedApp should be an ethereum address or a ENS name'
+          )
+        );
+      });
+    });
+
+    describe('When given authorizedUser is invalid', () => {
+      it('should throw a yup ValidationError with the correct message', async () => {
+        // --- GIVEN
+        const invalidAuthorizedUserAddress = '0x123456...';
+
+        await expect(
+          // --- WHEN
+          getGrantedAccess({
+            // @ts-expect-error No need for iexec here
+            iexec: {},
+            authorizedUser: invalidAuthorizedUserAddress,
+          })
+          // --- THEN
+        ).rejects.toThrow(
+          new ValidationError(
+            'authorizedUser should be an ethereum address or a ENS name'
+          )
+        );
+      });
+    });
+
+    describe('When given pageSize is invalid', () => {
+      it('should throw a yup ValidationError with the correct message', async () => {
+        // --- GIVEN
+        const pageSize = 9;
+
+        await expect(
+          // --- WHEN
+          getGrantedAccess({
+            // @ts-expect-error No need for iexec here
+            iexec: {},
+            page: 0,
+            pageSize,
+          })
+          // --- THEN
+        ).rejects.toThrow(
+          new ValidationError('pageSize must be greater than or equal to 10')
```

Comment:
```suggestion
          new ValidationError(`pageSize must be greater than or equal to ${pageSize+1}`)
```

Reply:
The minimum value (10) is hard-coded in the actual validation code:
```
  const vPageSize = numberBetweenSchema(10, 1000)
    .label('pageSize')
    .validateSync(pageSize);
```

So I don't see why not using it directly in the test code.

What is the idea you want to highlight with `${pageSize+1}`?